src/layout/layout.rs

```diff
@@ -2236,6 +2283,32 @@ mod tests {
             assert_eq!(expected, r);
         }
 
+        #[rstest]
+        #[case::length_overlap(vec![(0  , 20) , (20 , 20) , (40 , 20)] , vec![Length(20) , Length(20) , Length(20)] , Flex::Start        , 0)]
+        #[case::length_overlap(vec![(0  , 20) , (19 , 20) , (38 , 20)] , vec![Length(20) , Length(20) , Length(20)] , Flex::Start        , 1)]
+        #[case::length_overlap(vec![(21 , 20) , (40 , 20) , (59 , 20)] , vec![Length(20) , Length(20) , Length(20)] , Flex::Center       , 1)]
+        #[case::length_overlap(vec![(42 , 20) , (61 , 20) , (80 , 20)] , vec![Length(20) , Length(20) , Length(20)] , Flex::End          , 1)]
+        #[case::length_overlap(vec![(0  , 20) , (19 , 20) , (38 , 62)] , vec![Length(20) , Length(20) , Length(20)] , Flex::Legacy       , 1)]
+        #[case::length_overlap(vec![(0  , 34) , (33 , 34) , (66 , 34)] , vec![Length(20) , Length(20) , Length(20)] , Flex::SpaceBetween , 1)]
+        #[case::length_overlap(vec![(0  , 34) , (33 , 34) , (66 , 34)] , vec![Length(20) , Length(20) , Length(20)] , Flex::SpaceAround  , 1)]
```

Comment:
Try to use unique (distinct) test names here to make it possible to work out which test fails easily. The purpose of ::test_name is to change the test name from `case_01`, ... to `case_01_length_overlap.

Reply:
Done in https://github.com/ratatui/ratatui/pull/1398/commits/eff582110be19226c57e5778590cc7a99018e1aa